test/Microsoft.AspNetCore.Server.KestrelTests/TestHelpers/TestApplicationErrorLogger.cs

```diff
@@ -32,12 +34,17 @@ public void Log<TState>(LogLevel logLevel, EventId eventId, TState state, Except
             Console.WriteLine($"Log {logLevel}[{eventId}]: {formatter(state, exception)} {exception?.Message}");
 #endif
 
-            TotalErrorsLogged++;
+            TotalMessagesLogged++;
```

Comment:
Just remove this property, and use the now correct `TotalErrorsLogged` in the test that was using it.